app/routes/_index.tsx

```diff
@@ -41,6 +43,27 @@ export default function Index() {
           ))}
         </ul>
       </div>
+
+      <div className="py-5 grid justify-center text-center">
+        <h2 className="py-2 font-semibold text-3xl">What Region Am I In?</h2>
+        <p>
+          Aside from accrued time on the waitlist, distance is an important
+          factor. It's important to know what region you are in so you can see
+          how many others in your general area are waiting for the same organ.
+        </p>
+
+        <RegionStates region={1} />
```

Comment:
totally just fyi, this is a good use case for this:
https://www.joshwcomeau.com/snippets/javascript/range/
(This guy has great guides btw)

but i use this utility in a lot of my projects,
so i would do:

``` javascript
// 12 or use var for number of regions
{
  range(1, 12).map(n => <RegionStates region={n})
}
```